### ISSUE TITLE
Replace LiveData in Repository?

### ISSUE THREAD

**Issue by johngoren** [android/play-billing-samples#385]
Thanks for the wonderfully well-documented code sample. Just one thing: Word on the street is that we're not supposed to be using LiveData in Repositories anymore. Wonder what the Billing Repo would look like without it? Would we use Flow or something?

**Comment by calren** [comment]
That is correct that we don't recommend using LiveData in the Repository anymore! We are currently working on more guidance and samples around what this means in terms of alternative approaches.

**Comment by DAGalpin** [comment]
The TrivialDriveKotlin sample now uses Flow through the repository layer and then switches back to LiveData in the ViewModel using the asLiveData() extension.
